scripts/gqm_gen/README.md

```diff
@@ -2,11 +2,53 @@
 
 Creates a Mermaid diagram from Goal, Question, Metrics (GQM) Markdown files.
 
+The instructions below describe how to locally run 
+the script that builds the mermaid diagram. 
+A GitHub Actions will re-run this script at built time
+and regenerate the diagram, but it is good to run 
+locally to make sure it will build correctly.
+
+Be sure to also read the instructions in the 
+`measuring/use_gqm.md` file path.
+
+## Installation
+
+### Install mermaid CLI globally
+
+From this folder, follow instructions on the 
+[mermaid.js README](https://github.com/mermaid-js/mermaid-cli) to run in the terminal: 
+```
+npm install -g @mermaid-js/mermaid-cli
+```
+
+Install project specific node.js based on the package.json in this folder: 
+```
+npm install
+```
+
 ## Usage
 
 Builds the mermaid diagram and outputs it to stdout.
 
 ```bash
-npm install
-npm run --silent start
+npm run --silent start > gqm.mmd && mmdc --input gqm.md --output gqm.png
```

Comment:
Did you need to create an .mmd file to preview with your editor?

In vscode, it's possible to preview the .md file.

<img width="1068" alt="image" src="https://github.com/InnerSourceCommons/managing-innersource-projects/assets/776901/6222a019-a2a2-4261-9b50-d9baddef0f5c">


```suggestion 
npm run --silent start > gqm.md && mmdc --input gqm.md --output gqm.png
```